### ISSUE TITLE
After selecting "Specify path to hash" menu, I can't select another.

### ISSUE THREAD

**Issue by averov90** [FluxionNetwork/fluxion#1038]
### Small problem
In menu:
> A hash for the target AP was found.
> Do you want to use this file?
> 1) Use hash found
> 2) Specify path to hash
> 3) Rescan handshake directory

If I select the second item, I go to the path entry page.
I can enter the path, but I can go back by pressing Enter when the path is empty.

If I go back, then I am thrown to the menu for selecting the deauthentication method (mdk4, aireplay-ng, mdk3), which is already unexpected. However, when I reach the "Do you want to use this file?" Menu again, it does not appear, but instead navigates to the file path entry page.

That is, I have to either enter the path to the file, or restart fluxion to be able to select the "Use hash found" item.

### Bigger problem

Feedback from the captive portal site is unstable. I can’t say that it doesn’t work at all, because once it did work on the generic. However, it stopped working longer again. On sites that copy the interface of any routers (сhecked on ZyXEL and ARRIS_en interfaces), the feedback does not work at all. How many have not tested - sites do not react to either the correct hash or the wrong one, although the flyxion script itself reacts and stops the attack.

**Comment by jackblackjjx** [comment]
yeah, I meet the same thing like  you, I'm searched every *.cap,but still can not use its

**Comment by strasharo** [comment]
Please, retry the first reported issue with the latest version.
